app/routes/ui/scenario.py

```diff
@@ -62,6 +62,7 @@ async def get(self, scenario_friendly_name, scene_number=0):
             await self.render(
                 "scene.html",
                 is_last_scene=is_last_scene,
+                scene_id=scenario_obj.scene_ids[scene_number_int],
```

Comment:
nit: we are using `scenario_obj.scene_ids[scene_number_int]` in 3 places now (lines 41, 47, and 65), might be best to bring out to a variable and use that instead.